Added correct page linking

RFX14 opened the pull request:
Resolves #9

RFX14 committed:
Added correct page linking

calvh commented on app/__init__.py:
```diff
@@ -8,19 +8,36 @@
+    title = 'Projects'
+    profile_picture = './static/img/miyabi.jpg'
+    description = 'Project'
+    return render_template('projects.html', title=person, profile_picture=profile_picture , description=description, url=os.getenv("URL"))
```
You have `title = 'Projects'` but in your `return` statement you have `title=person`. Please change it to `title=title`

calvh commented on app/__init__.py:
```diff
@@ -8,19 +8,36 @@
+
+@app.route('/contact')
+def contact():
+    person = 'Contact'
```
Please change to `title=`Contact``

calvh commented on app/__init__.py:
```diff
@@ -8,19 +8,36 @@
+    person = 'Contact'
+    profile_picture = './static/img/miyabi.jpg'
+    description = 'contact'
+    return render_template('contact.html', title=person, profile_picture=profile_picture , description=description, url=os.getenv("URL"))
```
title=title

calvh commented on app/__init__.py:
```diff
@@ -8,19 +8,36 @@
+
+@app.route('/resume')
+def resume():
+    person = 'Resume'
```
`title='Resume'`

calvh commented on app/__init__.py:
```diff
@@ -8,19 +8,36 @@
+    person = 'Resume'
+    profile_picture = './static/img/miyabi.jpg'
+    description = 'Resume'
+    return render_template('resume.html', title=person, profile_picture=profile_picture , description=description, url=os.getenv("URL"))
```
`title=title`

calvh reviewed (changes requested):
Great work! Just some minor comments

calvh committed:
Fix render functions in __init__.py

Rename index.html to base.html
Modified layout of base.html
Add index.html as template for the About page
Add templates for contact, resume, projects pages
Fix CSS selector for ul tags

calvh commented:
Closes #2, #3, #5, #6, #9, #11, #12
